learnosity_sdk/_version.py

```diff
@@ -1 +1 @@
-__version__ = 'v0.3.8'
+__version__ = 'v0.3.9'
```

Comment:
Same, `release.sh` will take care of it.

Reply:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion. -->
**🔨 issue:**@shtrom @michael-linnane-lrn I actually just created a pre-release after merging my PR #80 but this used the `__version__.py` and overwrote `v0.3.8` on PyPI.

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously. -->
**🎯 suggestion:** If we bump the correct version here, it will be picked up by the release workflow created now by @walsh-conor and should deploy without issue from what I saw myself.